specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2022-12-27/HybridCompute.json

```diff
@@ -1858,6 +1993,465 @@
         }
       },
       "description": "Describes the subnet."
+    },
+    "ErrorDetail": {
```

Comment:
I *think* the breaking change notifications are happening because there is already an ErrorDetails defined as a common type, and the linter thinks (correctly or incorectly) that you're overriding it here. 

I think if you remove these and instead reference the common ErrorDetails and ErrorResponse types (search the json for ErrorDetails for examples) it will be happier, and because the issues are about new required properties it won't be an issue because these are only output types, not things the user submits.

Reply:
Done